src/fetch/types.ts

```diff
@@ -32,7 +32,7 @@ export namespace EdenFetch {
                 : {
                       method: Method
                   }) &
-            (IsNever<keyof Route['params']> extends true
```

Comment:
as `Route['params']` is already never, calling `keyof never` gets us `string | number | symbol` — js is weird